### ISSUE TITLE
API reference not really documenting

### ISSUE THREAD

**Issue by camilogarciabotero** [TidierOrg/Tidier.jl#132]
I was inspecting the latest deployed docs after #129 was merged and found that the API was empty. When built locally it shows all the API of the `rexport`ed packages. But it seems that in the deployed it doesn't. I believe this is on the `DocumenterVitepress.jl` side.

Proofs:

1. Locally:

![Screenshot 2024-04-09 at 4 11 31 PM](https://github.com/TidierOrg/Tidier.jl/assets/25586362/005a82bb-9a82-4761-86b9-e9b28a4d7691)

2. Deployment:

![Screenshot 2024-04-09 at 4 11 12 PM](https://github.com/TidierOrg/Tidier.jl/assets/25586362/dc0b427e-431a-4b01-a13f-ca5be8cbcad2)

**Comment by kdpsingh** [comment]
On one hand, from a user perspective, it would be really nice to have all the documentation in one place. So if there's a setting that would make re-exports show up on the generated docs, that would be great.

On the other hand, if it's not feasible, it's not a show-stopper as long as we link to the underlying base package documentation.

**Comment by camilogarciabotero** [comment]
Hi @lazarusA do you know if this is a bug or a not supported feature of `DocumenterVitepress.jl` or can we enable that on the deployment?
